src/endpoints/mex/mex.pair.service.ts

```diff
@@ -56,11 +58,11 @@ export class MexPairService {
     );
   }
 
-  async getMexPairsCount(filter?: MexPairExchange): Promise<number> {
+  async getMexPairsCount(filter?: MexPairsFilter): Promise<number> {
     const mexPairs = await this.getAllMexPairs();
 
-    if (filter) {
-      return mexPairs.filter(pair => pair.exchange === filter).length;
+    if (filter && filter.exchange) {
```

Comment:
should be a common function that applies the filter, to re-use the same logic when adding multiple filters